### DISCUSSION TITLE
Module parse failed: Unexpected token with Prisma package

### DISCUSSION THREAD

**Discussion by stephenasuncionDEV** [vercel/turborepo#3886] [Help] [answered]
I'm trying to make my database, Prisma, as an internal package in my own project.

I tried following this example: https://github.com/vercel/turbo/tree/main/examples/with-prisma/

However, when running the web app on dev I'm getting the following error:

![image](https://user-images.githubusercontent.com/73958774/220314523-2b2939b7-125b-4350-bfec-fd602ff7f6b2.png)

**Comment by rogix** [comment]
Are you using `transpilePackages` on your `next.config.js`? 

https://github.com/vercel/turbo/blob/main/examples/basic/apps/docs/next.config.js#L1

**Reply by stephenasuncionDEV** [reply]
Hi, not at all. Here is my next.config.js:

```ts
/** @type {import('next').NextConfig} */

/* eslint-disable-next-line  @typescript-eslint/no-var-requires */
const withBundleAnalyzer = require("@next/bundle-analyzer")({
  enabled: process.env.ANALYZE === "true",
});

const nextConfig = withBundleAnalyzer({
  reactStrictMode: false,
  swcMinify: true,
  env: {
    APP_NAME: process.env.APP_NAME,
    ...
  },
  images: {
    remotePatterns: [
      {
        protocol: "https",
        hostname: "**",
      },
    ],
    dangerouslyAllowSVG: true,
  },
});

module.exports = nextConfig;
```

**Comment by stephenasuncionDEV** [comment, marked as answer]
Weirdly enough, it started working now. Maybe I just needed to restart vscode and my computer.
